lib/generators/flipper/templates/migration.rb

```diff
@@ -12,12 +12,10 @@ def self.up
       t.string :value
       t.timestamps null: false
     end
-    add_foreign_key :flipper_gates, :flipper_features, on_delete: :cascade
```

Comment:
Add_foreign_key wasn't available in rails 4.1, I think the foreigner gem wasn't loading properly. 

What is the FK constraint accomplishing here that dependent destroy doesn't under normal operations? The tests pass without it. 

Cheers,
Carl Thuringer 

> On Aug 26, 2015, at 4:47 PM, Jesse Proudman <notifications@github.com> wrote:
> 
> In lib/generators/flipper/templates/migration.rb:
> 
> > @@ -12,12 +12,10 @@ def self.up
> >        t.string :value
> >        t.timestamps null: false
> >      end
> > -    add_foreign_key :flipper_gates, :flipper_features, on_delete: :cascade
> Reviewed the code and it doesn't look like I ever did need to touch these - they were working out of the box. What were they breaking for you?
> 
> ―
> Reply to this email directly or view it on GitHub.
>